app/views/restaurant/index.html.erb

```diff
@@ -1,15 +1,17 @@
-<h5>Dishley House</h5>
-<h2>What do you feel like tonight?</h2>
+<h2 style= "padding: 1em;">What do you feel like tonight?</h2>
 
 <div id="map"></div>
 
 <% @restaurants.group_by(&:category_name).each do |key, collection| %>
-  <div id="restaurant-category-<%= key.downcase %>">
+  <div id="restaurant-category-<%= key.downcase %>" style="padding: 1em;">
     <h1><%= key %></h1>
     <% collection.each do |restaurant| %>
       <div id="restaurant-<%= restaurant.id %>">
         <%= restaurant.name %>
-        <%= link_to 'Visit', restaurant_path(restaurant) %>
+        <button type="button" class="btn btn-success">
+          <%= button_to 'Visit', restaurant_path(restaurant), method: 'get' %>
```

Comment:
I would go with a link with button classes 
```
<%= link_to 'Visit', restaurant_path(restaurant), class: 'btn btn-success'  %>
```

Reply:
Working on that